product-service/handlers/getProductById/getProductById.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { APIGatewayProxyEvent, Context} from 'aws-lambda';
+import products from '../../mocks/products.json';
+import { getCORSHeaders } from '../../utils';
+import { errorMessages } from './errorMessages';
+import { getProductById } from '.';
+
+describe('getAllProducts handler', () => {
+  const context = {} as Context;
+  const callback = () => {};
+
+
+  it('should return a products by related id', async () => {
+    const result = await getProductById(
+      {
+        pathParameters: {
+          productId: '7567ec4b-b10c-48c5-9345-fc73c48a80aa',
+        },
+      } as unknown as APIGatewayProxyEvent,
```

Comment:
you may use `Partial<APIGatewayProxyEvent>` type for the event with only the needed properties